chia-protocol/src/program.rs

```diff
@@ -81,9 +75,71 @@ impl Program {
     }
 }
 
-impl Default for Program {
-    fn default() -> Self {
-        Self(vec![0x80_u8].into())
+impl From<Bytes> for Program {
```

Comment:
it's not obvious to me that a `Program` should look so much like a bytes buffer. Do you have specific use cases in mind for this?
My gut says conversions to and from buffers should be explicit

Reply:
No, I suppose `into_inner` is sufficient, I just want to have a way to go from owned `Program` to owned `Bytes`. I can remove the `From` impl if you prefer.

Reply:
I think that would make sense